token-lending/program/src/instruction.rs

```diff
@@ -31,257 +31,264 @@ pub enum LendingInstruction {
     ///
-    ///   0. `[writable]` Lending market account.
-    ///   1. `[]` Quote currency SPL Token mint. Must be initialized.
+    ///   0. `[writable]` Lending market account - uninitialized
+    ///   1. `[]` Quote currency SPL Token mint
     ///   2. `[]` Rent sysvar
-    ///   3. '[]` Token program id
+    ///   3. `[]` Token program id
     InitLendingMarket {
         /// Owner authority which can add new reserves
-        market_owner: Pubkey,
+        owner: Pubkey,
+        /// The target ratio of an obligation's borrows to deposits as a percent
+        loan_to_value_ratio: u8,
+        /// The percent at which an obligation is considered unhealthy
+        liquidation_threshold: u8,
     },
 
     // 1
+    /// Sets the new owner of a lending market.
+    ///
+    /// Accounts expected by this instruction:
+    ///
+    ///   0. `[writable]` Lending market account
+    ///   1. `[signer]` Current owner
+    SetLendingMarketOwner {
+        /// The new owner
+        new_owner: Pubkey,
+    },
+
+    // 2
     /// Initializes a new lending market reserve.
     ///
     /// Accounts expected by this instruction:
     ///
-    ///   0. `[writable]` Source liquidity token account.  $authority can transfer $liquidity_amount
+    ///   0. `[writable]` Source liquidity token account
+    ///                     $authority can transfer $liquidity_amount.
     ///   1. `[writable]` Destination collateral token account - uninitialized
-    ///   2. `[writable]` Reserve account.
+    ///   2. `[writable]` Reserve account - uninitialized
     ///   3. `[]` Reserve liquidity SPL Token mint
     ///   4. `[writable]` Reserve liquidity supply SPL Token account - uninitialized
-    ///   5. `[writable]` Reserve collateral SPL Token mint - uninitialized
-    ///   6. `[writable]` Reserve collateral token supply - uninitialized
-    ///   7. `[writable]` Reserve collateral fees receiver - uninitialized.
-    ///                     Owner will be set to the lending market account.
-    ///   8. `[]` Lending market account.
-    ///   9. `[signer]` Lending market owner.
-    ///   10 `[]` Derived lending market authority.
-    ///   11 `[]` User transfer authority ($authority).
+    ///   5. `[writable]` Reserve liquidity fee receiver - uninitialized
+    ///   6. `[writable]` Reserve collateral SPL Token mint - uninitialized
+    ///   7. `[writable]` Reserve collateral token supply - uninitialized
+    ///   8. `[]` Lending market account
+    ///   9. `[signer]` Lending market owner
+    ///   10 `[]` Derived lending market authority
+    ///   11 `[]` User transfer authority ($authority)
     ///   12 `[]` Clock sysvar
     ///   13 `[]` Rent sysvar
-    ///   14 '[]` Token program id
-    ///   15 `[optional]` Serum DEX market account. Not required for quote currency reserves. Must be initialized and match quote and base currency.
+    ///   14 `[]` Token program id
+    ///   15 `[optional]` Reserve liquidity aggregator account
```

Comment:
Cool, I'll break this up.

Reply:
I ended up rewriting the tests to use the new aggregator, and this changes the tests quite a bit. It would be a pain to extract into a separate PR, but after reviewing the reorganized changes, let me know if that's still needed.